### PR TITLE
Two new plot-generating scripts and changes in support of the ICLR poster.

### PR DESCRIPTION
**Pull request by ilyamironov**
+ plot_partition.py (budget allocation with smooth sensitivity).
+ utility_queries_answered.py (Figure 1, left).
+ several changes to the scripts required for the ICLR poster.

### CODE REVIEW

**Comment by ilyamironov** [comment]
LGTM

File: CODEOWNERS [hunk 1]
```diff
@@ -11,7 +11,7 @@
 /research/compression/ @nmjohn
 /research/deeplab/ @aquariusjay @yknzhu @gpapan
 /research/delf/ @andrefaraujo
-/research/differential_privacy/ @panyx0718 @mironov
+/research/differential_privacy/ @mironov @pseudorandom
```
Comment (karmel) [inline CODEOWNERS]: Is this the correct github uname? You appear to be @ilyamironov not mironov.

Comment (karmel) [reply CODEOWNERS]: And pseudorandom is probably not right either-- I suspect those are ldaps and not github usernames?

Comment (ilyamironov) [reply CODEOWNERS]: My bad. Resubmitting.

**Pull request closed without merging**